.github/workflows/ci.yml

```diff
@@ -23,15 +23,18 @@ jobs:
       # - run: npx nx-cloud start-ci-run --distribute-on="5 linux-medium-js" --stop-agents-after="e2e-ci"
 
       # Cache node_modules
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version-file: package.json
```

Comment:
cool

Reply:
it reads the version from volta property